lib/gemcutter/middleware/admin_auth.rb

```diff
@@ -0,0 +1,141 @@
+require_relative "../middleware"
+
+module GitHubOAuthable
```

Comment:
since this module is used in other controllers as well, can we pull it out to its own file?